Helper/ScopeSetting.php

```diff
@@ -143,6 +144,14 @@ public function getWebhookSecret($storeId = null)
         return $this->getScopeSetting(self::WEBHOOK_SECRET, $storeId);
     }
 
+    public function getWebhooks()
+    {
+        return $registeredWebhooks = [
+          ['product/delete', $this->getProductDeleteWebhookSetting()],
+          ['product/save', $this->getProductSaveWebhookSetting()],
+        ];
```

Comment:
Is there value in returning topics that aren't enabled? Thinking about other platforms for which we subscribe to certain webhooks (Shopify, WooCommerce), I'm pretty sure the endpoints that return this info only return the topics to which we are registered rather than a comprehensive list of all available.

I think it might be cleaner (and more easily extended in the future) to format this as an indexed array of associative arrays, similar to what those other platforms return. ex:
```
[
    [
        'topic' => 'product/delete',
        'enabled' => $this->getProductDeleteWebhookSetting(),
    ],
    [
        'topic' => 'product/save',
        'enabled' => $this-> getProductSaveWebhookSetting(),
    ]
]

Reply:
We could also allow for a param in this request to optionally fetch all available webhooks vs those that are enabled

Reply:
I'd be fine with only sending back only the enabled webhooks if we want to go that direction. In the app code that hits this endpoint I essentially filter out the ones that aren't enabled anyway. If we're only going to return the enabled ones then I'm not sure we need anything other than a list of the topics though?

Reply:
Yea, I guess since this is only for an inspector task it's probably fine however. Is there any other info stored about this? like when this setting was last updated for example? That could be useful for troubleshooting but may not be available

Reply:
Ahh, forgot about that. In that case if it's used in two different scenarios, I'd let the app code contain the logic to process the response/format the value to be saved and return as much info here for troubleshooting purposes which can be accessed via the inspector task.

Reply:
I'd love to see this return an indexed array containing an associative array for each webhook topic. Makes the logic in the app way easier to read instead of relying on the index for the topic and enabled boolean.

Reply:
did you see the updated logic for that method in the app? it still relies on the index but it unpacks the values into named variables which i think helps improve the readability.

```py
for webhook in webhook_config:
    webhook_name, is_enabled = webhook
    if int(is_enabled):
        registered_webhooks.append(webhook_name)
```

if i switch this back to an associative array, did you have logic in mind to pull out only the enabled webhooks' name into a list that would be more readable?

Reply:
sorry for the confusion around this! should be updated now though to what you were describing.